### DISCUSSION TITLE
ImportError: cannot import name parts_of_speech

### DISCUSSION THREAD

**Discussion by hiDaDeng** [explosion/spaCy#9992] [Help: Installation]
## How to reproduce the behaviour
just run ``import spacy`` in jupyter  and meet the ``ImportError: cannot import name parts_of_speech``

## Your Environment

* Operating System: Mac OS Catalina 10.15.3 
* Python Version Used: Anaconda 3.9
* spaCy Version Used:  3.2
* Environment Information:

**Comment by polm** [comment]
Can you please paste the full stack trace and the output of `conda list` and `pip freeze`? It sounds like something is wrong with your installation.

**Comment by Cofeecake** [comment]
Switch python environment then switch back to it. Or create a new venv.

**Reply by chenshi910814** [reply]
omg you are a genius!

**Reply by zhangjingze21** [reply]
He really is a miracle doctor.
